### PR TITLE
JANITORIAL: Fix another small bunch of typos

### PR DESCRIPTION
**Pull request by raziel-**
program.c and shader.c have typos in variables (actualy_deleteshader and actualy_detachshader)
But i think i caught all of them

feel free to cherry-pick

### CODE REVIEW

File: src/gl/program.c [hunk 1]
```diff
@@ -134,7 +134,7 @@ GLuint APIENTRY_GL4ES gl4es_glCreateProgram(void) {
     return program;
 }
 
-void actualy_deleteshader(GLuint shader);
+void actually_deleteshader(GLuint shader);
 void actualy_detachshader(GLuint shader);
```
Comment (ptitSeb) [inline src/gl/program.c:138]: missing the rename on `actually_detachsahder` declaration?

Comment (raziel-) [reply src/gl/program.c]: good catch, that one has escaped

**Comment by ptitSeb** [comment]
Thanks! (I do so many typos :( )

**Comment by raziel-** [comment]
heh, no problem...your repository is a fine test case for my typo checker script 😀

and thank you for merging

**Pull request merged**